frontend/packages/core/src/Contexts/workflow-storage-context/reducer.tsx

```diff
@@ -0,0 +1,74 @@
+import { removeLocalData, rotateDataFromAPI, storeLocalData } from "./helpers";
+import type { Action, BackgroundPayload, UserPayload, WorkflowStorageState } from "./types";
+
+/**
+ * Reducer for the WorkflowStorageContext
+ * This will act on the WorkflowStorageState and add / remove items from the temporary storage
+ * as well as localStorage, this will (optionally) keep all storage actions in one location for all
+ * workflows and allow for easier state hydration
+ */
+const workflowStorageContextReducer = (
+  state: WorkflowStorageState,
```

Comment:
I would rename this since it's a bit general right now to all storage pertaining to workflows

Reply:
Will do

Reply:
I thought this was pertaining to the original type name which I had modified to be `WorkflowStorageState`. Is the thought to rename 'state' to something else or the reducer?